api/people.js

```diff
@@ -9,14 +9,14 @@ router.use(utils.authMiddleware);
 
 router.get("/self", async (req, res) => {
   try {
-    const pointsProfile = await models.Person.findOrCreate({
+    const [person, isCreated] = await models.Person.findOrCreate({
```

Comment:
Where do we reference `isCreated`?

Reply:
we don't but findorcreate returns a pair that needs to be deconstructed.

Reply:
This is news to me

Reply:
Yeah we were returning the wrong value